content/telegraf/v1.18/administration/windows_service.md

```diff
@@ -8,37 +8,43 @@ menu:
     parent: Administration
 ---
 
-Telegraf natively supports running as a Windows service.
+Telegraf natively supports running as a Windows service. Complete the following steps:
 
-The following commands are available:
-
-| Command                            | Effect                        |
-|------------------------------------|-------------------------------|
-| `telegraf.exe --service install`   | Install telegraf as a service |
-| `telegraf.exe --service uninstall` | Remove the telegraf service   |
-| `telegraf.exe --service start`     | Start the telegraf service    |
-| `telegraf.exe --service stop`      | Stop the telegraf service     |
-
-Outlined below are the general steps to install Telegraf as a Service.
+1. Review considerations [before you begin](#before-you-begin).
+2. [Download and run as service](#download-and-run-as-service).
 
 {{% note %}}
-Installing a Windows service requires administrative permissions.
-Be sure to [launch Powershell as administrator](
+Installing a Windows service requires administrative permissions. [Launch PowerShell as administrator](
 https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7#with-administrative-privileges-run-as-administrator).
 {{% /note %}}
 
-1. Download the Telegraf binary and unzip its contents to `C:\Program Files\InfluxData\Telegraf`.
+## Before you begin
+
+The Telegraf installation includes an executable and configuration file. To simplify upgrades, consider the following:
+
+- **Setting up Windows symbolic link (Symlinks) to point to the Telegraf executable**. This option lets you easily roll back your upgrades, and maintain your custom configuration.
+- **Whether to use one or multiple Telegraf configurations**. You may want multiple configs if...
```

Comment:
something after the `...`?

Reply:
@kelseiv Should this be using "multiple Telegraf instances" to possibly direct users to information about running multiple Telegraf instances

Reply:
Thanks, @sjwang90! @pierwill plans to touch base with you on this one. Basically, in what use cases would users want to use multiple configs vs 1?

Reply:
And the second part...prob makes sense to add "multiple Telegraf instances" to the list of things to consider? To distinguish between when folks want to use multiple instances and when to use multiple configs?